styles.css

```diff
@@ -0,0 +1,133 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+.main {
+  background: url("images/IMG-20220819-WA0005.jpg");
+  background-repeat: no-repeat;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 100%;
+  font-family: Poppins, Arial, Verdana, sans-serif;
+}
+
+.top {
+  display: flex;
+  height: auto;
+  flex-direction: column;
+  margin-top: -15px;
+  background-size: auto;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.header {
+  margin: auto;
+  position: center;
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  font-family: Poppins, Arial, Verdana, sans-serif;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-grow: 0;
+  margin-left: 30px;
+  margin-right: 30px;
+}
+
+.logo {
+  font-family: Poppins, Arial, Verdana, sans-serif;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  line-height: 74px;
+  text-align: center;
+  vertical-align: center;
+  color: #6070ff;
+}
+
+a {
+  color: #6070ff;
+}
+
+.burger {
+  color: #6070ff;
+}
+
+.intro {
+  font-weight: bold;
+  margin-left: 24px;
+  margin-right: 24px;
+  width: 327px;
+  height: 150px;
+  top: 114px;
+  left: 24px;
+  font-size: 30px;
+  line-height: 52px;
+  margin-top: 100px;
+  align-items: left;
+  vertical-align: center;
+  color: #172b4d;
+}
+
+.about {
+  font-size: 16px;
+  margin-left: 24px;
+  margin-right: 24px;
+  width: 327px;
+  height: 146px;
+  top: 276px;
+  left: 24px;
+  line-height: 24px;
+  align-items: left;
+  vertical-align: top;
+  color: #344563;
+  margin-top: 40px;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.description {
+  font-style: medium;
+  font-size: 10px;
+  line-height: 24px;
+  margin-left: 24px;
+  margin-right: 225px;
+  align-items: left;
+  vertical-align: top;
+  color: #7f8cff;
+  margin-top: 16px;
+}
+
+.sidebar-toggle {
+  position: relative;
+  font-size: 1rem;
+  background: transparent;
+  border-color: transparent;
+  color: #6070ff;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.icons {
+  position: relative;
+  top: 0;
+  left: 0;
+  right: 24px;
```

Comment:
- [ ] Kindly remove the left space between icons. Doing that will improve the user interface a bit better.
your design:
![image](https://user-images.githubusercontent.com/91486791/185694564-9afd1236-cad9-419e-a17a-f2411cd23c87.png)
figma:
![image](https://user-images.githubusercontent.com/91486791/185694599-eec9c6cf-e326-4d5d-8995-9221c98cc01f.png)